orttraining/orttraining/python/orttraining_pybind_state.cc

```diff
@@ -330,6 +379,60 @@ void addObjectMethodsForTraining(py::module& m) {
         }
         ORT_THROW_IF_ERROR(static_cast<TrainingSession*>(sess->GetSessionHandle())->SetStateTensors(state_tensors, strict));
       })
+      .def("load_model_opt_state", [](PyTrainingSession* sess, std::unordered_map<std::string, py::object>& model_state, std::unordered_map<std::string, std::unordered_map<std::string, py::object>>& opt_state, bool strict) {
```

Comment:
may we use ```optimier``` instead of ```opt``` in the name since opt may sound like optional or may be misconstrued?

Reply:
Changed to `load_model_optimizer_state`